### DISCUSSION TITLE
What's the correct method to remove an extension?

### DISCUSSION THREAD

**Discussion by Acorn2Oak** [datenstrom/community#658] [Ask a question] [answered]
I tried one extension and decided against it.
My approach to rolling back was to remove all the files listed in the Extension ReadMe and delete yellow-system.ini entries.
Finally, I trawled through the latest file datetimes and zapped those that that were related.

Have I missed any automated approach?

**Comment by schulle4u** [comment, marked as answer]
You can also install, uninstall, and update extensions [from command line](https://github.com/annaesvensson/yellow-update). 

```
php yellow.php install gallery
php yellow.php install english german french
php yellow.php uninstall gallery
php yellow.php uninstall english german french
```

Remember to execute these commands with the appropriate owner rights, e. g. www-data.

**Reply by Acorn2Oak** [reply]
Most responsive.
I have a lot of digging to get into this method.

Thank you.
